docs/signature/README.md

```diff
@@ -0,0 +1,228 @@
+    ]
+  }
+}
+```
+
+### Signature Persisted within an OCI Artifact Enabled Registry
+
+Both values are persisted in a `signature.json` file. The file would be submitted to a registry as an Artifact with null layers.
+The `signature.json` would be persisted wthin the `manifest.config` object
+
+``` SHELL
+oras push \
+  registry.example.com/hello-world:v1.0 \
+  --manifest-config signature.json:application/vnd.cncf.notary.config.v2+json
+```
+
+Would push the following manifest:
+
+``` JSON
+{
+  "schemaVersion": 2,
+  "config": {
+    "mediaType": "application/vnd.cncf.notary.config.v2+json",
+    "size": 1906,
+    "digest": "sha256:c7848182f2c817415f0de63206f9e4220012cbb0bdb750c2ecf8020350239814"
+  },
+  "layers": []
+}
+```
+
+## *Signature* Property Descriptions
+
+- **`signed`** *object*
+
+  This REQUIRED property provides the signed content.
+
+  - **`iat`** *integer*
+
+    This OPTIONAL property identities the time at which the manifests were presented to the notary. This field is based on [RFC 7519 Section 4.1.6](https://tools.ietf.org/html/rfc7519#section-4.1.6). When used, it does not imply the issue time of any signature in the `signatures` property.
+
+  - **`nbf`** *integer*
+
+    This OPTIONAL property identifies the time before which the  signed content MUST NOT be accepted for processing. This field is based on [RFC 7519 Section 4.1.5](https://tools.ietf.org/html/rfc7519#section-4.1.5).
+
+  - **`exp`** *integer*
+
+    This OPTIONAL property identifies the expiration time on or after which the signed content MUST NOT be accepted for processing. This field is based on [RFC 7519 Section 4.1.4](https://tools.ietf.org/html/rfc7519#section-4.1.4).
+
+  - **`digest`** *string*
+
+    This REQUIRED property is the *digest* of the target manifest, conforming to the requirements outlined in [Digests](https://github.com/opencontainers/image-spec/blob/master/descriptor.md#digests). If the actual content is fetched according to the *digest*, implementations MUST verify the content against the *digest*.
+
+  - **`size`** *integer*
+
+    This REQUIRED property is the *size* of the target manifest. If the actual content is fetched according the *digest*, implementations MUST verify the content against the *size*.
+
+  - **`references`** *array of strings*
```

Comment:
In the meeting we discussed signature options other than x509. If these are used, how will this domain name verification be done?

Reply:
in a word: "dunno". 
It's an interesting concept that enables a number of additional, but not required scenarios.
Let's say I publish an image from wabbit-networks.io If the CN=registry.wabbit-networks.io and the referenced artifacts have the same registry name, I can simply know where they originated from. In theory, even go back to that registry as I may have gotten the images from an aggregator (docker.io), or my private registry.
More importantly, when I pull the registry.wabbit-networks.io/net-monitor:v1 image into my private repository for my production deployment, I can scan it, test it, verify it works in my acme-rockets environment. I then add another signature to the image, which is now at acme-rockets.azurecr.io/net-monitor:v1. That additional signature has the new registry url on it. My production k8s environment can be configured with OPA (Open Policy Agent) to assure that all content is signed by the acme-rockets key, and all content must be signed under the acme-rockets.azurecr.io registry and pulled from this registry. 
So, it can provide extra validation/policy enforcement.
Now, this is just a riff that Shiwei came up with using x509. I wouldn't say it's a hard constraint. Think of it as a interesting tile pattern on this mocked bathroom we've been designing. It's not as important as having the bidet (that Cormack and Ralph Squillace say are requirements) . It's an idea we can keep or toss. Remember, we're sketching things out to see what we like and don't like.